generator/index.js

```diff
@@ -32,6 +32,7 @@ module.exports.handler = async (event, context) => {
     const ext = require('path').extname(filePath)
     switch (ext) {
       case '.html':
+      case '':
```

Comment:
Smart idea! I don't get how stripping the HTML works 🤔 will CF see it as a directory and serve it like it has `index.html`?

Reply:
Stripping is done in the other file in this PR. I think CF first checks for a matching file and if not found will treat it as a folder.